go-controller/pkg/types/const.go

```diff
@@ -140,4 +140,14 @@ const (
 	OVSDBWaitTimeout = 0
 
 	ClusterLBGroupName = "clusterLBGroup"
+
+	// Default Meters created on GRs.
```

Comment:
should we move this to L101 under other Meter related constants?

Reply:
Do you mean that we have a better place for meter related constants already or to create a new one?
This current place `types/const.go` is very non-specific and not very big so I think it fits ok.
Would you rather have something like `types/meter_const.go`? or a different package altogether? If so, what would be the package?
Perhaps, if we see the need of reorganizing the constants we could have a follow up PR.

Reply:
I meant we already have
```
	// Meter constants
	OvnACLLoggingMeter   = "acl-logging"
	OvnRateLimitingMeter = "rate-limiter"
	PacketsPerSecond     = "pktps"
	MeterAction          = "drop"
```
let's just move this up to that part.